### PR TITLE
dead links updated for scaling and rollups

### PR DESCRIPTION
**Pull request by kwsorensen**
Fixing broken links on pages.

## Description

I was looking into eth2 merge details and noticed there was broken links on a few pages.

## Related Issue
https://github.com/ethereum/ethereum-org-website/issues/3393

### CODE REVIEW

**Comment by kwsorensen** [comment]
@corwintines @GeorgeTrotter  @jjmstark Can I get a review?

File: src/content/translations/de/eth2/shard-chains/index.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Wenn die ersten Shard Chains implementiert sein werden, werden diese zunächst e
 
 Rollups sind eine "Layer 2"-Technologie, die heute bereits existiert. Sie erlauben dApps, Transaktionen zu bündeln oder sie mittels "Roll up" in eine einzelne Transaktions-Off-Chain zu vereinen, einen kryptographischen Beweis zu erstellen und diesen dann zur Chain zu senden. Dies reduziert die Datenmenge, die für eine Transaktion benötigt wird. Kombiniert mit der zusäzlichen Datenverfügbarkeit durch die Shards erhält man 100.000 Transaktionen pro Sekunde.
 
-[Mehr zu Rollups](/developers/docs/layer-2-scaling/)
+[Mehr zu Rollups](/developers/docs/scaling/layer-2-rollups/#rollups)
```
Comment (wackerow) [inline src/content/translations/de/eth2/shard-chains/index.md:51]: Not sure we need to direct to the #rollups header here. We already redirect `/docs/layer-2-scaling/` to `/docs/layer-2-rollups/` automatically, I would just stick with that for these links, ie:
```
[Mehr zu Rollups](/developers/docs/scaling/layer-2-rollups/)
```

File: src/content/translations/hr/eth2/shard-chains/index.md [hunk 2]
```diff
@@ -48,15 +48,15 @@ Kada se isporuče prvi lanci djelića, oni će samo pružiti dodatne podatke mre
...
 
 Plan je uvijek bio dodati dodatnu funkcionalnost djelićima, kako bi postali sličniji današnjoj [ glavnoj mreži Ethereum ](/glossary/#mainnet). To bi omogućilo pohranu i izvršavanje pametnih ugovora i upravljanje računima. No, s obzirom na pojačanje u broju transakcija u sekundi koje pružaju djelići verzije 1, je li to potrebno? O tome se još uvijek raspravlja u zajednici i čini se da postoji nekoliko mogućnosti.
 
 ### Trebaju li djelići izvršenje koda? {#do-shards-need-code-execution}
 
-Vitalik Buterin, u podcastu Bankless, predstavio je 3 potencijalne opcije o kojima vrijedi razgovarati. <iframe width="100%" height="315" src="/developers/docs/layer-2-scaling/" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen mark="crwd-mark"></iframe>
+Vitalik Buterin, u podcastu Bankless, predstavio je 3 potencijalne opcije o kojima vrijedi razgovarati. <iframe width="100%" height="315" src="/developers/docs/scaling/#layer-2-scaling" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen mark="crwd-mark"></iframe>
```
Comment (wackerow) [inline src/content/translations/hr/eth2/shard-chains/index.md:59]: Looks like this was a bug previously, but the `src` here should be the YouTube embed link: https://www.youtube.com/embed/-R0j5AMUSzA?start=5841
```
Vitalik Buterin, u podcastu Bankless, predstavio je 3 potencijalne opcije o kojima vrijedi razgovarati. <iframe width="100%" height="315" src="https://www.youtube.com/embed/-R0j5AMUSzA?start=5841" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen mark="crwd-mark"></iframe>
```

File: src/content/translations/ro/history/index.md [hunk 3]
```diff
@@ -140,16 +140,16 @@ Furculița Byzantium:
...
 [Citește anunțul Fundației Ethereum](https://blog.ethereum.org/2017/10/12/byzantium-hf-announcement/)
 
 <ExpandableCard title="EIP-uri Byzantium" contentPreview="Official improvements included in this fork.">
 
 - [EIP-140](https://eips.ethereum.org/EIPS/eip-140) – _adaugă opcodul „REVERT”._
 - [EIP-658](https://eips.ethereum.org/EIPS/eip-658) – _adaugă câmpul „stare” la recepțiile tranzacției pentru a indica succesul sau eșecul_
-- [EIP-196](https://eips.ethereum.org/EIPS/eip-196) – _adaugă curba eliptică și înmulțirea scalară pentru a permite [ZK-Snarks](/developers/docs/layer-2-scaling/#rollups)._
-- [EIP-197](https://eips.ethereum.org/EIPS/eip-197) – _adaugă curba eliptică și înmulțirea scalară pentru a permite [ZK-Snarks](/developers/docs/layer-2-scaling/#rollups)._
+- [EIP-196](https://eips.ethereum.org/EIPS/eip-196) – _adaugă curba eliptică și înmulțirea scalară pentru a permite [ZK-Snarks](/developers/docs/scaling/layer-2-rollups/#rollups-and-sidechains)._
+- [EIP-197](https://eips.ethereum.org/EIPS/eip-197) – _adaugă curba eliptică și înmulțirea scalară pentru a permite [ZK-Snarks](/developers/docs/scaling/layer-2-rollups/#rollups-and-sidechains)._
```
Comment (wackerow) [inline src/content/translations/ro/history/index.md:152]: `#rollups-and-sidechains` does not exist as a header link. The existing internal link of `#rollups` should be used here.

File: src/content/translations/ro/history/index.md [hunk 4]
```diff
@@ -140,16 +140,16 @@ Furculița Byzantium:
 - A redus recompensele [miniere](/developers/docs/consensus-mechanisms/pow/mining/) pe bloc de la 5 la 3 ETH.
 - A amânat [bomba de dificultate](/glossary/#difficulty-bomb) cu un an.
 - A adăugat capacitatea de a efectua apeluri către alte contracte care nu modifică starea.
-- A adăugat anumite metode de criptografie pentru a permite \[scalarea layer-ului 2\]((/developers/docs/layer-2-scaling/).
+- A adăugat anumite metode de criptografie pentru a permite \[scalarea layer-ului 2\]((/developers/docs/scaling/#layer-2-scaling).
```
Comment (wackerow) [inline src/content/translations/ro/history/index.md:143]: Similar to others, would change a;; `/layer-2-scaling/` to the main `scaling/layer-2-rollups` page:
```
/developers/docs/scaling/layer-2-rollups/
```

File: src/content/translations/ro/history/index.md [hunk 5]
```diff
@@ -95,7 +95,7 @@ Furculița Istanbul:
 - [EIP-1108](https://eips.ethereum.org/EIPS/eip-1108) –_criptografie mai ieftină pentru a îmbunătăți costurile de [gas](/glossary/#gas)._
 - [EIP-1344](https://eips.ethereum.org/EIPS/eip-1344) – _protejează Ethereum împotriva atacurilor de reluare prin adăugarea „CHAINID” [opcode](/developers/docs/ethereum-stack/#ethereum-virtual-machine)._
 - [EIP-1884](https://eips.ethereum.org/EIPS/eip-1884) – _optimizarea prețurilor gazului opcode pe baza consumului._
-- [EIP-2028](https://eips.ethereum.org/EIPS/eip-2028) – _reduce costul CallData pentru a permite mai multe date în blocuri – bun pentru [scalarea Layer 2](/developers/docs/layer-2-scaling/)._
+- [EIP-2028](https://eips.ethereum.org/EIPS/eip-2028) – _reduce costul CallData pentru a permite mai multe date în blocuri – bun pentru [scalarea Layer 2](/developers/docs/scaling/#layer-2-scaling)._
```
Comment (wackerow) [inline src/content/translations/ro/history/index.md:98]:
```
/developers/docs/scaling/layer-2-rollups/
```

File: src/content/translations/ro/history/index.md [hunk 6]
```diff
@@ -83,7 +83,7 @@ Furculița Istanbul:
 
 - A optimizat costul [gazului](/glossary/#gas) anumitor acțiuni din [EVM](/developers/docs/ethereum-stack/#ethereum-virtual-machine).
 - A îmbunătățit rezistența la atacul prin refuz de serviciu.
-- A făcut soluțiile de [scalare Layer 2](/developers/docs/layer-2-scaling/) bazate pe SNARK-uri și STARK-uri mai performante.
+- A făcut soluțiile de [scalare Layer 2](/developers/docs/scaling/#layer-2-scaling) bazate pe SNARK-uri și STARK-uri mai performante.
```
Comment (wackerow) [inline src/content/translations/ro/history/index.md:86]:
```
/developers/docs/scaling/layer-2-rollups/
```

File: src/content/translations/ro/eth2/shard-chains/index.md [hunk 7]
```diff
@@ -48,7 +48,7 @@ Când vor fi livrate primele lanțuri de fragmente, acestea vor furniza doar dat
 
 Rollup-urile sunt o tehnologie de „nivel 2” care există astăzi. Acestea permit aplicațiilor descentralizate (dapps) să grupeze („roll up”) tranzacții într-o singură tranzacție în afara lanțului, să genereze o dovadă criptografică și apoi să o trimită lanțului. Acest lucru reduce datele necesare executării unei tranzacții. Combină acest lucru cu disponibilitatea tuturor datelor suplimentare oferită de fragmente și obții 100.000 de tranzacții pe secundă.
 
-[Mai multe despre rollup-uri](/developers/docs/layer-2-scaling/)
+[Mai multe despre rollup-uri](/developers/docs/scaling/layer-2-rollups/#rollups-and-sidechains)
```
Comment (wackerow) [inline src/content/translations/ro/eth2/shard-chains/index.md:51]:
```
/developers/docs/scaling/layer-2-rollups/
```

File: src/content/translations/ro/enterprise/index.md [hunk 8]
```diff
@@ -71,7 +71,7 @@ Diverse organizații au depus eforturi comune pentru a face Ethereum Enterprise
...
+[Nivelul 2](/developers/docs/scaling/#layer-2-scaling) este un set de tehnologii sau sisteme care rulează peste Ethereum (Nivelul 1), moștenesc proprietăți de securitate din Nivelul 1 și oferă o capacitate mai mare de procesare a tranzacțiilor (transfer), taxe de tranzacție mai mici (cost de operare) și confirmări mai rapide ale tranzacțiilor decât Nivelul 1. Soluțiile de scalare Nivel 2 sunt securizate de Nivelul 1, dar permit aplicațiilor blockchain să gestioneze mai mulți utilizatori sau acțiuni sau date decât ar putea găzdui Nivelul 1. Multe dintre ele valorifică progresele recente în criptografie și dovezile de cunoaștere-zero (ZK) pentru a maximiza performanța și securitatea.
```
Comment (wackerow) [inline src/content/translations/ro/enterprise/index.md:74]:
```
/developers/docs/scaling/layer-2-rollups/
```

**Review by wackerow — CHANGES_REQUESTED** [review]
Hey @kwsorensen, thanks for helping update these! Left some notes, but in general I think the `/layer-2-scaling/ links should just be changed to `/scaling/layer-2-rollups/`. 

Also noted a youtube embed link that got screwed up at some point, and a header id (#rollups-and-sidechains) that doesn't exist, if you wouldn't mind touching these up.

**Comment by kwsorensen** [comment]
@wackerow I think your requests should be fixed.

**Review by kwsorensen — COMMENTED** [review]
I believe the changes you requested are satisfied.

File: src/content/translations/hi/eth2/shard-chains/index.md [hunk 9]
```diff
@@ -48,7 +48,7 @@ summaryPoints:
 
 रोलअप एक "लेयर 2" तकनीक है, जो आज भी मौजूद है। वे एक एकल लेनदेन ऑफ-चेन में लेनदेन को बंडल या "रोल अप" करने की अनुमति देते हैं, एक क्रिप्टोग्राफिक प्रमाण उत्पन्न करते हैं और फिर इसे चेन में जमा करते हैं। इससे लेनदेन के लिए आवश्यक डेटा कम हो जाता है। इसे सभी अतिरिक्त डेटा उपलब्धता के साथ साझा करें, जो कि शार्ड द्वारा दी गई है और आपको प्रति सेकंड 100,000 लेनदेन मिलते हैं।
 
-[रोलअप के बारे में अधिक जानकारी](/developers/docs/layer-2-scaling/)
+[रोलअप के बारे में अधिक जानकारी](/developers/docs//docs/scaling/layer-2-rollups/#rollups)
```
Comment (samajammin) [inline src/content/translations/hi/eth2/shard-chains/index.md:51]:
```suggestion
[रोलअप के बारे में अधिक जानकारी](/developers/docs/docs/scaling/layer-2-rollups/#rollups)
```

**Review by samajammin — APPROVED** [review]
Thanks for the updates!

**Comment by samajammin** [comment]
@all-contributors please add @kwsorensen for content.

**Pull request merged**